icaruscode/Filters/FilterDataIntegrity_module.cc

```diff
@@ -0,0 +1,96 @@
+////////////////////////////////////////////////////////////////////////
+//
+// FilterDataIntegrity class
+//
+////////////////////////////////////////////////////////////////////////
+#include <fstream>
+
+/// Framework includes
+#include "art/Framework/Core/ModuleMacros.h"
+#include "art/Framework/Core/EDFilter.h"
+#include "art/Framework/Principal/Event.h"
+#include "canvas/Utilities/InputTag.h"
+#include "fhiclcpp/ParameterSet.h"
+#include "messagefacility/MessageLogger/MessageLogger.h"
+#include "lardataobj/RecoBase/Hit.h"
```

Comment:
Definitely not needed:
```suggestion
```

Reply:
Yeah, guilty as charged. Cloned from the filter module that tests on TPC hits.